docs/index.md

```diff
@@ -1,21 +1,31 @@
 # Welcome to Anubis
 
-Anubis is a flexible Policy Enforcement solution that makes easier to reuse
-security policies across different services, assuming the policies entail the
-same resource.
+Anubis is a flexible Policy Enforcement solution
+that makes easier to reuse security policies across different services,
+assuming the policies entail the same resource.
+In short we are dealing with policy portability :) What do you mean by policy
```

Comment:
"In short, we are dealing with policy portability.

What do we mean by it? Let's think of a user..."